app/utilities/decimal_places.py

```diff
@@ -40,29 +40,18 @@ def custom_format_currency(
     parsed_locale = Locale.parse(locale)
```

Comment:
This relates to the comment above: https://github.com/ONSdigital/eq-questionnaire-runner/pull/1148#discussion_r1301523857. I was using the locale delimiter initially but it breaks because by the time we reach this method the delimiter will always have been formatted and will be `.`